include/boost/geometry/algorithms/detail/envelope/linear.hpp

```diff
@@ -0,0 +1,99 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2007-2015 Barend Gehrels, Amsterdam, the Netherlands.
+// Copyright (c) 2008-2015 Bruno Lalande, Paris, France.
+// Copyright (c) 2009-2015 Mateusz Loskot, London, UK.
+
+// This file was modified by Oracle on 2015.
+// Modifications copyright (c) 2015, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Menelaos Karavelas, on behalf of Oracle
+
+// Parts of Boost.Geometry are redesigned from Geodan's Geographic Library
+// (geolib/GGL), copyright (c) 1995-2010 Geodan, Amsterdam, the Netherlands.
```

Comment:
I have now omitted these lines in many files in `envelope` and `expand`.
I have to admit it is not clear for me when to keep it and when not.

The general guideline I was following was to keep it as long as: (1) I consider a file as originating (fully or partially) from an older one (even if the new file lies at a different location and has different name), and (2) the old file was having these two lines referring to Geodan.